newrelic.cjs

```diff
@@ -0,0 +1,32 @@
+exports.config = {
```

Comment:
Don't exclude files from `xo`. If there's a problem with a specific rule, you can always disable it for one line or for the whole file. Here, put `/* eslint-disable camelcase */` at the top.